package/kernel/mac80211/patches/rt2x00/602-04-wifi-rt2x00-Support-EEPROM-swap-binding.patch

```diff
@@ -13,27 +13,23 @@ Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
 
 --- a/drivers/net/wireless/ralink/rt2x00/rt2x00eeprom.c
 +++ b/drivers/net/wireless/ralink/rt2x00/rt2x00eeprom.c
-@@ -20,6 +20,19 @@
+@@ -20,6 +20,15 @@
  #include "rt2x00soc.h"
  
  #if IS_ENABLED(CONFIG_MTD)
 +static void rt2800lib_eeprom_swap(struct rt2x00_dev *rt2x00dev)
 +{
-+	struct device_node *np = rt2x00dev->dev->of_node;
 +	size_t len = rt2x00dev->ops->eeprom_size;
 +	int i;
 +
-+	if (!of_find_property(np, "ralink,eeprom-swap", NULL))
-+		return;
```

Comment:
This check is useful for ralink target. It seems that lantiq target using the big endian byte order.